src/router/kromo/dd-wrt/wikar/style.css

```diff
@@ -229,4 +229,4 @@ input.spaceradio, .submitFooter input.button {
     margin-bottom: 2px;
     width: 100px;
   }
-}
```

Comment:
if you read down to here, see how many unnecessary `px` units are being removed from `0` values? surely this is less NOT more file size? Again it is not necessary. it maybe was one million years ago.

It is relevant, Im removing unnecessary  crud, I dont see how removing unecessary stuff adds to ANY file size.